### PR TITLE
Add and Load `ovm1` data to V2

### PR DESCRIPTION
**Pull request by chuxinh**
Adding `ovm1` data as sql files to add to Dune V2. Some files are quite large that I got large file warnings :p 

*For Dune Engine V2*
I've checked that:
General checks:
* [x] I tested the query on dune.com after compiling the model with dbt compile (compiled queries are written to the target directory)
* [x] I used "refs" to reference other models in this repo and "sources" to reference raw or decoded tables 
* [ ] if adding a new model, I added a test
* [x] the filename is unique and ends with .sql
* [x] each sql file is a select statement and has only one view, table or function defined  
* [x] column names are `lowercase_snake_cased`
* [ ] if adding a new model, I edited the dbt project YAML file with new directory path for both models and seeds (if applicable)
* [x] if adding a new model, I edited the alter table macro to display new database object (table or view) in UI explorer
* [x] if adding a new materialized table, I edited the optimize table macro

Join logic:
* [ ] if joining to base table (i.e. ethereum transactions or traces), I looked to make it an inner join if possible

Incremental logic:
* [ ] I used is_incremental & not is_incremental jinja block filters on both base tables and decoded tables
  * [ ] where block_time >= date_trunc("day", now() - interval '1 week')
* [ ] if joining to base table (i.e. ethereum transactions or traces), I applied join condition where block_time >= date_trunc("day", now() - interval '1 week')
* [ ] if joining to prices view, I applied join condition where minute >= date_trunc("day", now() - interval '1 week')

### CODE REVIEW

**Review by soispoke — CHANGES_REQUESTED** [review]
What's currently missing from this PR is a ovm1_schema.yml file that contains the tags, column descripitons, model names, etc...
Much like this file here but for all models:
https://github.com/duneanalytics/spellbook/pull/1529/files#diff-75dad4f33d458e82aa08edf723de059f6daab2855317f8ca4c9f5c864fe1c7a8

File: models/ovm1/optimism/ovm1_optimism_schema.yml [hunk 1]
```diff
@@ -0,0 +1,206 @@
...
+  - name: ovm1_optimism_contracts
+    meta:
+      blockchain: optimism
+      project: ovm1
+      contibutors: MSilb7, chuxinh
+    config:
+      tags: ["optimism", "ovm1", "contracts"]
+    description: "Contracts deployed on OVM1."
+    columns:
+      - &name: contract_address
```
Comment (soispoke) [inline models/ovm1/optimism/ovm1_optimism_schema.yml:13]: - &name: contract_name

should be replaced by 

- &contract_name
        name: contract_name

in all & instances in this file

**Review by soispoke — CHANGES_REQUESTED** [review]

**Review by soispoke — CHANGES_REQUESTED** [review]
Hi @chuxinh sorry for the late feedback (as you know we're all preparing for DuneCon), here are two errors that need fixing before I can approve this PR:
- The number of columns defined in `ovm1_optimism_contracts` doesn't match the number of values 
- There is a `nan` as an input in `ovm1_optimism_q1_user_address_daily_transactions` that isn't accepted

**Comment by soispoke** [comment]
Error screenshot: 
Capture d’écran 2022-09-14 à 08 41 40

**Comment by chuxinh** [comment]
> Hi @chuxinh sorry for the late feedback (as you know we're all preparing for DuneCon), here are two errors that need fixing before I can approve this PR:
> 
> * The number of columns defined in `ovm1_optimism_contracts` doesn't match the number of values
> * There is a `nan` as an input in `ovm1_optimism_q1_user_address_daily_transactions` that isn't accepted

Thanks for checking! Should be fixed for those now

**Review by soispoke — APPROVED** [review]
thanks @chuxinh, lgtm

**Pull request merged**